### ISSUE TITLE
🐛 biome json schema for 2.4.8 missing new drizzle nursery rules

### ISSUE THREAD

**Issue by machadinhos** [biomejs/biome#9545]
### Environment information

<details>

```bash

Paste output here

```

</details>

### What happened?

Hi!

The new drizzle nursery linting rules seem to be missing from the json schema.

To reproduce put this in your config:
```json
{
  "$schema": "https://biomejs.dev/schemas/2.4.8/schema.json",
  "linter": {
    "enabled": true,
    "rules": {
      "nursery": {
        "noDrizzleUpdateWithoutWhere": "on",
        "noDrizzleDeleteWithoutWhere": "on"
      }
    }
  }
}
```

You get this warning: `Property noDrizzleUpdateWithoutWhere is not allowed.`

I checked the schema on https://biomejs.dev/schemas/2.4.8/schema.json and they are actually missing.

### Expected result

The rules have been added to the official docs with `Rule available since: v2.4.8`:
https://biomejs.dev/linter/rules/no-drizzle-update-without-where/

No errors/warnings should appear in the biome config file and the rules should be added to the schema.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by machadinhos** [comment]
I didn't actually test running biome lint, but even with the rules setup on the `biome.json` config it is not detecting the error/info when I run `biome lint`. Maybe the rules weren't added to the release of biome 2.4.8 and in that case the docs and release notes are wrong.

Btw, I love biome, I don't wanna sound entitled or something 😅

**Comment by Netail** [comment]
A fix in the website repo has been merged, the schema will be live shortly :)
